### PR TITLE
fix(youtube): unthemed text and icons

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title
image
image

also this is probably going to be my last PR before i stop contributing lol

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/youtube/catppuccin.user.css [hunk 1]
```diff
@@ -502,6 +505,10 @@
     [fill="white"] {
       fill: if(@lookup = latte, @base, @text) !important;
     }
+     
+    [src*="https://www.gstatic.com/images/icons/material/system/1x/check_circle_grey600_36dp.png"] {
```
Comment (uncenter) [inline styles/youtube/catppuccin.user.css:509]: Can you add a comment here with more details.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/youtube/catppuccin.user.css [hunk 2]
```diff
@@ -61,7 +61,9 @@
     color: @text;
     background: @base !important;
     color-scheme: if(@lookup = latte, light, dark);
-
+    
+    @text-filter: @catppuccin[@@lookup][@text_filter];
```
Comment (isabelroses) [inline styles/youtube/catppuccin.user.css:65]: can you put this with the rest of the lookup table

File: styles/youtube/catppuccin.user.css [hunk 3]
```diff
@@ -980,6 +988,18 @@
...
+      color: @text !important;
+    }
+    
+    .profile-badge-view-model-wiz__badge-subtitle {
+      color: @subtext0 !important;
+    }
+    
+    .yt-profile-identity-info-view-model-wiz__divider {
+      border-color: @surface0 !important;
+    }
```
Comment (isabelroses) [inline styles/youtube/catppuccin.user.css:1002]: can you leave a comment for what this is

**Review by isabelroses — CHANGES_REQUESTED** [review]

**Review by isabelroses — APPROVED** [review]

**Review by uncenter — APPROVED** [review]

**Comment by somerand0mcat** [comment]
btw @uncenter, can i contact you on discord?
(its not going to end like last time)

**Comment by uncenter** [comment]
About what?

**Comment by somerand0mcat** [comment]
> About what?

the pronouns.cc PR

**Comment by uncenter** [comment]
> > About what?
> 
> the pronouns.cc PR

What about it can't be discussed on the PR itself?

**Comment by somerand0mcat** [comment]
> What about it can't be discussed on the PR itself?
i mean i was just going to ask what should we do about the placeholder situation

**Comment by uncenter** [comment]
There is literally a thread for that exact topic. As I said *in that thread*, I will look more closely when I have a chance to review.

**Pull request merged**